.github/workflows/ci.yml

```diff
@@ -23,6 +23,7 @@ env:
   MAVEN_FAST_INSTALL: "-B --strict-checksums -V --quiet -T C1 -DskipTests -Dmaven.source.skip=true -Dair.check.skip-all"
   MAVEN_GIB: "-P gib -Dgib.referenceBranch=refs/remotes/origin/${{ github.event.pull_request.base.ref }}"
   MAVEN_TEST: "-B --strict-checksums -Dmaven.source.skip=true -Dair.check.skip-all --fail-at-end -P gib -Dgib.referenceBranch=refs/remotes/origin/${{ github.event.pull_request.base.ref }}"
+  MAVEN_COMPILE_COMMITS: "-B --strict-checksums --no-snapshot-updates --no-transfer-progress -T C1 -DskipTests -Dmaven.javadoc.skip=true -Dmaven.source.skip=true -Dair.check.skip-all=true -P gib -Dgib.referenceBranch=refs/remotes/origin/${{ github.event.pull_request.base.ref }} -pl '!:trino-server-rpm'"
```

Comment:
How about adding `--quiet`? I think we should only be interested in errors, not the regular output, especially if it's going to be so similar for multiple commits.